GDJS/Runtime/runtimeobject.ts

```diff
@@ -2135,6 +2224,19 @@ namespace gdjs {
      */
     static forcesGarbage: Array<gdjs.Force> = [];
 
+    /**
+     * Arrays and data structure that are (re)used by RuntimeObject.separateFromObjects to
+     * avoid any allocation.
+     * @static
+     */
+    private static separateFromObjectsStatics: {
```

Comment:
I think we can even extract this... outside of the class! :) 
That would avoid us to expose this member on the class interface